src/osd/PrimaryLogPG.cc

```diff
@@ -8178,6 +8212,9 @@ int PrimaryLogPG::_rollback_to(OpContext *ctx, ceph_osd_op& op)
 
       if (obs.exists) {
 	t->remove(soid);
+	if (obs.oi.has_manifest()) {
+	  dec_all_refcount_manifest(obs.oi, ctx);
```

Comment:
Where do we handle the new head chunk_map duplicated from the clone?

Reply:
I added a commit to update object_manifest_t also.

Reply:
Does this indicate a missing test?